### PR TITLE
DOC: update 1.13 release note for MaskedArray, masked constants ellipisis indexing

### LINKED ISSUE #9121: numpy masked array regression in 1.13.0rc1
With version 1.12.1 the following script

```python
from numpy import ma, __version__
print __version__
x = ma.array(1, mask=True)
print x[...],type(x[...])
```

produced

```
1.12.1
-- <class 'numpy.ma.core.MaskedConstant'>
```

With 1.13.0rc1 I get

```
1.13.0rc1
-- <class 'numpy.ma.core.MaskedArray'>
```

This means that `x[...] is ma.masked` returns `False` in 1.13, when it used to return `True`.

So it appears that the behaviour of scalar masked arrays has changed.  This is causing test failures in [netcdf4-python](https://github.com/Unidata/netcdf4-python).

### PR DESCRIPTION
**Pull request by ahaldane**
Fixes #9121

(backport coming up)

### CODE REVIEW

**Comment by eric-wieser** [comment]
Surely "wraps", not "unwraps". Don't forget that this also changes `arr[i,...]` on 1d arrays

**Comment by ahaldane** [comment]
@eric weiser, updated. Does this version sound accurate?

File: doc/release/1.13.0-notes.rst [hunk 1]
```diff
@@ -137,6 +137,11 @@ invokes``ndarray.__getslice__`` (e.g. through ``super(...).__getslice__``) will
 now issue a DeprecationWarning - ``.__getitem__(slice(start, end))`` should be
 used instead.
 
+Indexing MaskedArrays/Constants with ... (ellipsis) now returns a MaskedArray
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (charris) [inline doc/release/1.13.0-notes.rst]: Use dashes, '-', not `~` and put the ellipsis in quotes.

**Comment by ahaldane** [comment]
OK, updated as you suggested, and changed the commit title to be DOC too.

File: doc/release/1.13.0-notes.rst [hunk 2]
```diff
@@ -137,6 +137,11 @@ invokes``ndarray.__getslice__`` (e.g. through ``super(...).__getslice__``) will
 now issue a DeprecationWarning - ``.__getitem__(slice(start, end))`` should be
 used instead.
 
+Indexing MaskedArrays/Constants with ... (``ellipsis``) now returns MaskedArray
```
Comment (eric-wieser) [inline doc/release/1.13.0-notes.rst]: That should be `Ellipsis`, not `ellipsis`. I think @charris' comment was requesting ` ``...`` `

Comment (ahaldane) [inline doc/release/1.13.0-notes.rst]: Oh! that makes more sense

**Comment by charris** [comment]
Looks good, thanks Allan.

**Pull request merged**